api-guidelines/rest/http/headers/rules/may-use-test-header.md

```diff
@@ -0,0 +1,43 @@
+---
+id: R0000XX
+---
+
+# MAY use `Test` header
+
+API consumers and providers may use the OTTO-specific test header described within this rule to communicate that a request or response is a result of a test in a **non-live** environment. This enables testing certain use cases together with multiple teams while simultaneously enabling all teams to identify and ignore hindering test data.
+
+Requests and responses **must not** be flagged with `Test` in the live environment as this could negatively impact the data integrity of the overall system.
```

Comment:
This part I took from the async version of this rule because I thought we had the same assumptions about having no test interactions in a live environment. But you are right. If we indeed have tests in a live environment, it would be beneficial to flag those interactions with test headers. Do you have the API performance tests in mind? Does it happen in a live environment as well?
 
I will change the sentence afterwards.

Reply:
Hey @holgerb, we removed the sentence about test header usage in live environment because, as you mentioned, it has no real benefit of leaving them in live.

Reply:
Solved for me!